examples/react-client/fishjam-chat/src/components/Tile.tsx

```diff
@@ -1,30 +1,49 @@
 import VideoPlayer from "./VideoPlayer";
 import { Track } from "@fishjam-cloud/react-client";
+import AudioVisualizer from "./AudioVisualizer.tsx";
 
 type Props = {
   id: string;
   name: string;
   videoTrack: Track | undefined;
+  audioTrack?: Track | undefined;
```

Comment:
**nitpick:** `audioTrack?: Track | undefined;` is the same as `audioTrack?: Track;`,

but `audioTrack?: Track | undefined;` is not the same as `audioTrack: Track | undefined;`, the latter one will accept `undefined` as a value but will require to pass a value in the props